python-lists-and-tuples/solution.py

```diff
@@ -0,0 +1,22 @@
+L = 'abcdefghijklmnopqrstuvwxyz'
+
+def create_combine_two(input):

```

Comment:
The first function should just return the two lists. Then the combining of the lists should be done either in its own function or just after the first function definition.